templates/ui.tera

```diff
@@ -0,0 +1,29 @@
+---
+whiskers:
+  version: 2.5.0
+  filename: "ui-theme.md"
+---
+{% macro bgr(c) -%}
+{{ (c.rgb.b * 65536) + (c.rgb.g * 256) + c.rgb.r }}
+{%- endmacro bgr %}
+# Changing UI Colors
+Notepad++ has limited support for customizing the UI elements.
+Some elements are controlled by the Windows theme (menu dropdowns, scrollbars, etc.) and can't be changed.
+
+You can theme the supported elements by modifying a line in the file `%AppData%/Notepad++/config.xml`.
+Replace the line beginning with `<GUIConfig name="DarkMode">` with the desired flavor.
+
+> [!NOTE]
+> You cannot use Npp to make this change, as it will overwrite any changes to `config.xml` when closing.
```

Comment:
Nitpick but let's use the fullname instead of abbreviating it.

```suggestion
> You cannot use Notepad++ to make this change, as it will overwrite any changes to `config.xml` when closing.
```